app/assets/stylesheets/components/find/_call-out-box.scss

```diff
@@ -0,0 +1,19 @@
+.govuk-inset-text.app-callout {
+  background-color: #f3f2f1;
+  padding: 15px;
+  border-left-width: 5px;
+  min-height: 130px;
+
+  &.app-callout--pink {
+    border-left-color: #d43681;
+  }
+
+  @include govuk-media-query($from: mobile) {
+    min-height: auto;
+  }
```

Comment:
I believe `mobile` is the smallest supported viewport size. In that case, you can remove this and line 5 entirely. The tablet and above should take effect only from screen sizes of "tablet" and above.